### PR TITLE
Fixed typos on README and improved CONTRIBUTING description

### PR DESCRIPTION
**Pull request by hudsonsferreira**
Just fixed typos on README and improved some descriptions at CONTRIBUTING.

### CODE REVIEW

**Comment by bf4** [comment]
Looks good. in the future you can include `[ci skip]` in your commit for documentation only changes

**Comment by hudsonsferreira** [comment]
Thanks for the tip @bf4 :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ Rake can be found at http://docs.seattlerb.org/rake/.
 
 To run a single test suite
 
-   rake test TEST=path/to/test.rb
+`rake test TEST=path/to/test.rb`
```
Comment (beauby) [inline CONTRIBUTING.md]: We might as well add a prefixing `$`, what do you think?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ Rake can be found at http://docs.seattlerb.org/rake/.
...
 
-which can be further narrowed down to one test:
+Which can be further narrowed down to one test:
 
-   rake test TEST=path/to/test.rb TESTOPTS="--name=test_something"
+`rake test TEST=path/to/test.rb TESTOPTS="--name=test_something"`
```
Comment (beauby) [inline CONTRIBUTING.md]: Same here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (hudsonsferreira) [inline CONTRIBUTING.md]: I agree :)

**Comment by hudsonsferreira** [comment]
I'm sorry about forget `[ci skip]` at last commit.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bf4) [inline CONTRIBUTING.md]: I personally don't like the $ since I can't copy and paste it

Comment (hudsonsferreira) [inline CONTRIBUTING.md]: I took a look in all bash code that we have at .md before answer and saw that they always have $.

Comment (hudsonsferreira) [inline CONTRIBUTING.md]: On the other hand @bf4 $ make explicit the ideia that is bash command instead of ruby, for instance.

Comment (beauby) [inline CONTRIBUTING.md]: The `$` makes it clear it's a shell command. Plus, nobody should blindly copy-paste shell commands.

Comment (hudsonsferreira) [inline CONTRIBUTING.md]: :+1:

Comment (bf4) [inline CONTRIBUTING.md]: > The $ makes it clear it's a shell command. Plus, nobody should blindly copy-paste shell commands.

I'd rather copy a command than retype it, but I guess I'm not blind :)

**Comment by beauby** [comment]
:+1: LGTM. Could you squash your commits into one before I merge?

**Comment by hudsonsferreira** [comment]
@beauby make sense squash both context into one commit? or squash just which make README reference?

**Comment by hudsonsferreira** [comment]
@beauby and @bf4 could you please check it again? :)

**Comment by beauby** [comment]
LGTM, merging. Thanks @hudsonsferreira

**Pull request merged**